CoffeeChat/Controller/InterestsViewController.swift

```diff
@@ -147,7 +130,33 @@ class InterestsViewController: UIViewController {
                 switch result {
                 case .value(let response):
                     if response.success {
-                        // TODO: Fix interest model in backend before using to populate screen
+                        self.selectedInterests = response.data.interests.map {
+                            // TODO: Fix interest model in backend before using to populate screen
+                            return Interest(name: $0, categories: [], imageURL: nil)
+                        }
+                        self.updateNext()
+                        self.fadeTableView.view.reloadData()
+                    }
+                case .error(let error):
+                    print(error)
+                }
+            }
+        }
+    }
+    
+    private func getAllInterests() {
+        NetworkManager.shared.getAllInterests().observe { [weak self] result in
+            guard let self = self else { return }
+            DispatchQueue.main.async {
+                switch result {
+                case .value(let response):
+                    if response.success {
```

Comment:
id replace the if `response.success { <indented code> }` with a guard let that prints an error message, so the code that maps the interest and reloads the tableview normally isn't indented as much

so something like
```
guard response.success else {
    print("some error message")
    return
}
self.interests = ...
tableview.reloadData()
```

Reply:
also kinda curious -- what happens if this isn't a success? (aka what's the behavior if we fail to get the interests -- or groups, goals, etc.  on the other screens)